### PR TITLE
Added Docker environment with OpenVINO

### PR DESCRIPTION
**Pull request by asmorkalov**

### CODE REVIEW

File: ubuntu-github-actions-openvino--20.04/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Version: 20230724
+# Image name: quay.io/opencv-ci/opencv-ubuntu-20.04-openvino
+
+FROM openvino/ubuntu20_dev
```
Comment (dkurt) [inline ubuntu-github-actions-openvino--20.04/Dockerfile]: Should we apply a specific tag? `FROM openvino/ubuntu20_dev:2023.0.0` or  even `FROM openvino/ubuntu20_dev:2023.1.0`?

Comment (mshabunin) [reply ubuntu-github-actions-openvino--20.04/Dockerfile]: > Should we apply a specific tag?

I agree, we should use version tag.

Comment (asmorkalov) [reply ubuntu-github-actions-openvino--20.04/Dockerfile]: Agree. Done!

**Comment by asmorkalov** [comment]
@dkurt @mshabunin I fixed OpenVINO version 2023.0.2. Build with 2023.1.0 produce lot of warnings about nGraph deprecation and some test failures.

**Pull request merged**